test/i18n_test.rb

```diff
@@ -287,6 +287,34 @@ def setup
     assert_equal I18n.config.available_locales.size * 2, I18n.config.available_locales_set.size
   end
 
+  test "interpolation_keys returns an array of keys" do
+    store_translations(:en, "example_two" => "Two interpolations %{foo} %{bar}")
+    assert_equal ["foo", "bar"], I18n.interpolation_keys("example_two")
+  end
+
+  test "interpolation_keys returns an empty array when no interpolations " do
+    store_translations(:en, "example_zero" => "Zero interpolations")
+    assert_equal [], I18n.interpolation_keys("example_zero")
+  end
+
+  test "interpolation_keys returns an empty array when missing translation " do
+    assert_equal [], I18n.interpolation_keys("does-not-exist")
```

Comment:
This _could_ raise an exception or return `nil`, but in the interest of simplicity and consistency, I feel it's probably fine to return `[]`.

Reply:
I think `[]` is better here as it makes the return type consistently `Array` rather than `Array | nil`